code/modules/projectiles/guns/projectile/mosin.dm

```diff
@@ -2,7 +2,8 @@
 	name = "mosin nagant"
 	desc = "JOY OF HAVING MOSIN NAGANT RIFLE IS JOY THAT MONEY CANNOT AFFORD. "
 	fire_sound = 'sound/weapons/mosin.ogg'
-	icon_state = "mosin"
+	icon = 'icons/obj/mosinbig.dmi' //36x32 file to fit.
```

Comment:
:thinking: why not just gunsbig or something

Reply:
Also why 36x32 instead of 64x64 or 48x48

Reply:
The issue with that is sprites scale at 1:1, so a 64x64 sprite would take up 4 whole turfs. It's 36x32 so I had a few more pixels to work with to get the proportions looking alright. This way it only sticks onto the next turf by 3-4px.

Reply:
The issue with that is sprites scale at 1:1, so a 64x64 sprite would take up 4 whole turfs. It's 36x32 so I had a few more pixels to work with to get the proportions looking alright. This way it only sticks onto the next turf by 3-4px.

Reply:
Add it `biggun.dmi`, which is 48x32 for some reason.

Reply:
>sprite would take up 4 whole turfs.

That's not how it works. It literally doesn't matter if the dmi icon size is 32x32 or 128x128. Bar signs dmi is 128x128 yet most signs are 32x64